.gitignore

```diff
@@ -19,6 +19,7 @@ hs_err_pid*
 build/
 classes/
 dsb-integration/out/
+dsb-gui/out/*
```

Comment:
Instead of this, remove this line and the one above it and replace it with `out/*`